crates/core/tedge/src/cli/certificate/c8y/download.rs

```diff
@@ -144,13 +144,13 @@ impl DownloadCertCmd {
             };
 
             // Read the security token from /dev/tty
-            let security_token = if self_security_token.is_empty() {
-                rpassword::read_password_from_tty(Some("Enter security token: "))?
+            let one_time_password = if self_one_time_password.is_empty() {
+                rpassword::read_password_from_tty(Some("Enter one-time-password: "))?
```

Comment:
```suggestion
                rpassword::read_password_from_tty(Some("Enter one-time password: "))?
```

Reply:
Fixed in 04d94c76a95b56b02d6f1f14c7b1029e415c19e4